rskj-core/src/test/java/co/rsk/cli/RskCliTest.java

```diff
@@ -62,6 +61,14 @@ void argsAreParsedCorrectly() {
         assertEquals(6, parsedArgs.getFlags().size());
         assertEquals(2, parsedArgs.getOptions().size());
         assertEquals(1, parsedArgs.getParamValueMap().size());
+
```

Comment:
I would move this whole block so it's located just after the testnet args. WDYT?

Reply:
Solved